beakerx_tabledisplay/js/src/dataGrid/store/datastore/bplustree.ts

```diff
@@ -0,0 +1,1525 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2018, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
```

Comment:
The repo is using [`Apache License 2.0`](https://github.com/twosigma/beakerx_tabledisplay/blob/master/LICENSE). I'm not sure if BSD would be compatible here?

Reply:
It is not a problem to include a BSD-licensed code in Apache License 2.0 code: https://www.apache.org/legal/resolved.html#category-a